Saudox/app/Http/Controllers/Profissional/ProfissionalEvolucaoController.php

```diff
@@ -82,6 +82,12 @@ public function editarJudo($id_paciente) {
 
     // NEUROPSICOLOGIA
     public function verNeuropsicologia($id_paciente) {
+
+        $profissional_logado = Auth::user();
+        if(!$profissional_logado->podeCriarEvolucao()) {
```

Comment:
mas esse é o método de ver e não de criar

Reply:
Eu errei o nome do método chamado. O correto seria:
```suggestion
        if(!$profissional_logado->podeCriarPaciente()) {
```

Reply:
Conferido o nome do método a ser atualizado
![image](https://user-images.githubusercontent.com/25617786/97577348-1454ef00-19ce-11eb-9c42-b1a2bf53470b.png)